apps/website/assets/styles/_variables.scss

```diff
@@ -90,4 +90,14 @@
 
   // SHADOWS
   --shadow-primary: 0px 4px 25px -5px rgba(0, 0, 0, 0.25);
+
```

Comment:
Hi @smjsmsam , this looks really great, thank you!
The only thing I would recommend is to remove css variables below from _variables.scss since they are only used for header as far as I get. 
You can just use the values like #f9f9f9  in the header file styles and if later on some of these will be needed in other parts of the app we will create the variables. Thanks!